content/en/docs/concepts/scheduling-eviction/resource-bin-packing.md

```diff
@@ -3,70 +3,98 @@ reviewers:
 - bsalamat
 - k82cn
 - ahg-g
-title: Resource Bin Packing for Extended Resources
+title: Resource Bin Packing
 content_type: concept
 weight: 80
 ---
 
 <!-- overview -->
 
-{{< feature-state for_k8s_version="v1.16" state="alpha" >}}
-
-The kube-scheduler can be configured to enable bin packing of resources along
-with extended resources using `RequestedToCapacityRatioResourceAllocation`
-priority function. Priority functions can be used to fine-tune the
-kube-scheduler as per custom needs. 
+In the [scheduling-plugin](/docs/reference/scheduling/config/#scheduling-plugins)  `NodeResourcesFit` of kube-scheduler, there are two 
+scoring strategies that support the bin packing of resources: `MostAllocated` and `RequestedToCapacityRatio`.
 
 <!-- body -->
 
-## Enabling Bin Packing using RequestedToCapacityRatioResourceAllocation
+## Enabling bin packing using MostAllocated strategy
+The `MostAllocated` strategy scores the nodes based on the utilization of resources, favoring the ones with higher allocation.
+For each resource type, you can set a weight to modify its influence in the node score.
+
+To set the `MostAllocated` strategy for the `NodeResourcesFit` plugin, use a [scheduler configuration](https://kubernetes.io/docs/reference/scheduling/config) similar to the following:
```

Comment:
Wrap long lines properly; use path rather than full URL for links.

```suggestion
To set the `MostAllocated` strategy for the `NodeResourcesFit` plugin, use a
[scheduler configuration](/docs/reference/scheduling/config) similar to the following:
```

Reply:
Done